_targets.R

```diff
@@ -103,6 +104,8 @@ window_length <- 20  ##needs to be <= complete_years
 increment <- 1
 min_yrs_in_window<- 15  ##minimum number of years of data required within a window
 min_windows <- 10  ##Must have this many windows available in order to plot 
+###percentage of drainage area overlap between nested basins above which they will be grouped together
+nested_threshold<-.5
```

Comment:
Okay, sounds good! I think that makes sense. And because we have few gages that are nested within each region, I am not concerned about testing different thresholds right now. I think our time can be better used working on other tasks